maps/eaz_comparison_tool.py

```diff
@@ -257,4 +262,4 @@ def update_plots(selected_variable, selected_dimension, selected_year):
 
 # Run the app
 if __name__ == "__main__":
-    app.run_server(debug=True)
+    app.run_server(debug=False, host="0.0.0.0", port=8050)
```

Comment:
This is more related to the actual deployment, but I did see this warning when running the app in the container. I know you mentioned that the final form of this app is somewhat up in the air so I'm all about getting it out to the right people on the right timeline. Either way, I wanted to mention it - we don't have a lot of experience deploying Flask apps outside of AWS Lambdas.

`WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.`